### PR TITLE
Clarify usage of flag for dual genomes

### LINKED ISSUE #47: To clarify the usage of the flag for dual genomes
## What analysis module should be updated and why?

To clarify the usage of the flag for dual genomes.

### PR DESCRIPTION
**Pull request by AntoniaChroni**
# Purpose/implementation Section

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] This change requires a documentation update.

## What scientific question/module of analysis is your analysis addressing?

To clarify the usage of the flag for dual genomes.

## What GitHub issue does your pull request address?

Closes #47.

## Directions for reviewers 
Tell potential reviewers what kind of feedback you are soliciting.

### Which areas should receive a particularly close look?

Please review for clarity and test-run the `cellranger-analysis` module for the single genome: `genome_name: "GRCm39"`.

## Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

File: analyses/cellranger-analysis/j2.sh [hunk 1]
```diff
@@ -28,5 +28,4 @@ mkdir -p ./results/03_cellranger_count_summary
 ########################################################################
 # Run to summarize results generated by CellRanger for all libraries
 python ./util/summarize_cellranger_results.py --dir=./results/02_cellranger_count/${cellranger_parameters} \
-                                              --outdir=./results/03_cellranger_count_summary/ \
-                                              --genome ${genome_name_cellranger}
\ No newline at end of file
+                                              --outdir=./results/03_cellranger_count_summary/
```
Comment (sharonfreshour) [inline analyses/cellranger-analysis/j2.sh:31]: Maybe consider leaving the --genome flag within the script, but commented out (similar to how the --force_cells flag is left in the j1.sh script), to make it a bit simpler for the user to edit

**Review by sharonfreshour — APPROVED** [review]
Test run completed successfully and generated the QC_Summary_CellRanger_Report.tsv from the j2 job. 

I did leave one comment regarding leaving the --genome flag in the j2.sh script, but merging should be good to go either way

File: analyses/cellranger-analysis/j2.sh (hunk 1 above)
Comment (sharonfreshour) [reply analyses/cellranger-analysis/j2.sh:31]:
```suggestion
                                              --outdir=./results/03_cellranger_count_summary/
                                              # --genome ${genome_name_cellranger}
```

Comment (AntoniaChroni) [reply analyses/cellranger-analysis/j2.sh]: See e71701a6671ae2b83fbb04835c926f2167c96b59

**Pull request merged**